_config.yml

```diff
@@ -4,7 +4,7 @@
 # Make our URLs look nice. We're not a blog with dated posts, we just want the title.
 permalink:     /:title.html
 
-archive: false
+archive: true
```

Comment:
`archive` gets overwritten in the build.sh on the master branch. I' prefer we do that instead of committing this change.